NuGet.config

```diff
@@ -1,5 +1,6 @@
 <configuration>
     <config>
         <add key="repositoryPath" value="packages" />
+        <add key="dotnet-tools" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json">
```

Comment:
I think you forgot to add `</add> in the end of this line, or perhaps a `/` before ending the tag

Reply:
Thanks!